src/js/_enqueues/wp/theme.js

```diff
@@ -118,9 +119,11 @@ themes.view.Appearance = wp.Backbone.View.extend({
 		// Render and append after screen title.
 		view.render();
 		this.searchContainer
+			.find('.search-box')
 			.append( $.parseHTML( '<label for="wp-filter-search-input">' + l10n.search + '</label>' ) )
-			.append( view.el )
-			.on( 'submit', function( event ) {
+			.append( view.el );
+
+		this.searchContainer.on( 'submit', function( event ) {
 				event.preventDefault();
 			});
```

Comment:
```suggestion
			event.preventDefault();
		});
```

These two lines do not need as much indentation.